Only in curl-7.20.1/lib: .Makefile.vc8.swp
diff -u -r curl-7.20.1-orig/lib/Makefile.vc8 curl-7.20.1/lib/Makefile.vc8
--- curl-7.20.1-orig/lib/Makefile.vc8	2010-04-14 07:56:19.000000000 -0700
+++ curl-7.20.1/lib/Makefile.vc8	2010-06-11 16:50:01.741850000 -0700
@@ -101,6 +101,7 @@
 SSLLIBS    = libeay32.lib ssleay32.lib
 ZLIBLIBSDLL= zdll.lib
 ZLIBLIBS   = zlib.lib
+ZLIBLIBSDBG= zlibd.lib
 WINLIBS    = wsock32.lib bufferoverflowu.lib wldap32.lib
 CFLAGS     = $(CFLAGS)
 
@@ -289,7 +290,7 @@
 TARGET   = $(LIB_NAME_DEBUG).lib
 DIROBJ   = $(CFG)
 LFLAGSZLIB = "/LIBPATH:$(ZLIB_PATH)"
-LNK      = $(LNKLIB) $(ZLIBLIBS) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
+LNK      = $(LNKLIB) $(ZLIBLIBSDBG) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
 CC       = $(CCDEBUG) $(RTLIBD) $(CFLAGSZLIB) $(CFLAGSLIB)
 CFGSET   = TRUE
 !ENDIF
@@ -314,7 +315,7 @@
 DIROBJ   = $(CFG)
 LFLAGSZLIB = "/LIBPATH:$(ZLIB_PATH)"
 LFLAGSSSL = "/LIBPATH:$(OPENSSL_PATH)\out32"
-LNK      = $(LNKLIB) $(ZLIBLIBS) $(LFLAGSSSL) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
+LNK      = $(LNKLIB) $(ZLIBLIBSDBG) $(LFLAGSSSL) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
 CC       = $(CCDEBUG) $(RTLIBD) $(CFLAGSSSL) $(CFLAGSZLIB) $(CFLAGSLIB)
 CFGSET   = TRUE
 !ENDIF
diff -u -r curl-7.20.1-orig/lib/Makefile.vc9 curl-7.20.1/lib/Makefile.vc9
--- curl-7.20.1-orig/lib/Makefile.vc9	2010-04-14 07:56:19.000000000 -0700
+++ curl-7.20.1/lib/Makefile.vc9	2010-06-11 16:50:04.175450000 -0700
@@ -101,6 +101,7 @@
 SSLLIBS    = libeay32.lib ssleay32.lib
 ZLIBLIBSDLL= zdll.lib
 ZLIBLIBS   = zlib.lib
+ZLIBLIBSDBG= zlibd.lib
 WINLIBS    = ws2_32.lib wldap32.lib
 CFLAGS     = $(CFLAGS)
 
@@ -289,7 +290,7 @@
 TARGET   = $(LIB_NAME_DEBUG).lib
 DIROBJ   = $(CFG)
 LFLAGSZLIB = "/LIBPATH:$(ZLIB_PATH)"
-LNK      = $(LNKLIB) $(ZLIBLIBS) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
+LNK      = $(LNKLIB) $(ZLIBLIBSDBG) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
 CC       = $(CCDEBUG) $(RTLIBD) $(CFLAGSZLIB) $(CFLAGSLIB)
 CFGSET   = TRUE
 !ENDIF
@@ -314,7 +315,7 @@
 DIROBJ   = $(CFG)
 LFLAGSZLIB = "/LIBPATH:$(ZLIB_PATH)"
 LFLAGSSSL = "/LIBPATH:$(OPENSSL_PATH)\out32"
-LNK      = $(LNKLIB) $(ZLIBLIBS) $(LFLAGSSSL) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
+LNK      = $(LNKLIB) $(ZLIBLIBSDBG) $(LFLAGSSSL) $(LFLAGSZLIB) /out:$(DIROBJ)\$(TARGET)
 CC       = $(CCDEBUG) $(RTLIBD) $(CFLAGSSSL) $(CFLAGSZLIB) $(CFLAGSLIB)
 CFGSET   = TRUE
 !ENDIF
